src/I18n/RelativeTimeFormatter.php

```diff
@@ -99,7 +99,7 @@ public function diffForHumans(
      */
     public function timeAgoInWords(I18nDateTimeInterface $time, array $options = []): string
     {
-        $options = $this->_options($options, FrozenTime::class);
```

Comment:
Should we retain an alias for `FrozenTime` so that users don't have to churn a bunch of code for the class rename? Or is that something that we should add to rector rules for 5.x?

Reply:
It would be a simple rename so we could just add rector rules for it.

Reply:
I've added it to the transform list in the 5.x wiki.